src/main/java/com/openlattice/search/SearchService.java

```diff
@@ -292,25 +295,23 @@ public void deleteOrganization( OrganizationDeletedEvent event ) {
     @Subscribe
     public void indexEntities( EntitiesUpsertedEvent event ) {
         UUID entityTypeId = dataModelService.getEntityTypeByEntitySetId( event.getEntitySetId() ).getId();
-        elasticsearchApi.createBulkEntityData( entityTypeId, event.getEntitySetId(), event.getEntities() );
-    }
-
-    private void indexLinkedEntities(
-            UUID linkingEntitySetId, Map<UUID, Set<UUID>> linkingIds, Map<UUID, PropertyType> propertyTypes ) {
-        if ( !linkingIds.isEmpty() ) {
-            UUID entityTypeId = dataModelService.getEntityTypeByEntitySetId( linkingEntitySetId ).getId();
+        if ( elasticsearchApi.createBulkEntityData( entityTypeId, event.getEntitySetId(), event.getEntities() ) ) {
+            // mark them as indexed
+            indexingMetadataManager.markAsIndexed(
```

Comment:
Wrap this in a timer. We need visibility to make sure thread pool doesn't get backlogged by users making small writes.

It should be distinct from the timer as markAsIndexed as it will skew results from background indexing service unless we break out aggregation of all markAsIndexed. @UnsungHero97